src/utils/identity.test.ts

```diff
@@ -0,0 +1,95 @@
+import {SubstrateChainIdentityJudgement as IdentityJudgement} from 'src/generated/litentryGraphQLTypes';
+import {mapStatusText} from './identity';
+
+test('isErroneous test case', () => {
```

Comment:
You can use [each](https://jestjs.io/docs/api#describeeachtablename-fn-timeout) for the entire file so you don't have to repeat the same for each data set

Reply:
I'll keep this in mind, but in this case, we have different expected outputs for two other cases rewriting using `describe.each (...)` providing multiple objects will make it complicated.